tools/seriallog.cc

```diff
@@ -1,4 +1,5 @@
 #include <exception>
+#include <functional>
```

Comment:
It should have always been there but I guess it gets pulled in with clang and gcc versions < 7.